composer.json

```diff
@@ -15,12 +15,14 @@
   },
   "autoload": {
     "psr-4": {
-      "OpenTelemetry\\Contrib\\Aws\\": "src/Aws/src",
+      "OpenTelemetry\\Aws\\": "src/Aws/src",
```

Comment:
This one looks odd, I think that it should still contain `Contrib` in the namespace...

Reply:
Agreed, I was updating it to match the subpackage composer.json namespace, which I think needs updating: https://github.com/open-telemetry/opentelemetry-php-contrib/blob/872475913cfc30684bc302a0b74890a5897cee9b/src/Aws/composer.json#L36 
(that's what my question at the end of the PR comment was about)

Reply:
Should I update those here or in separate PRs?